web_environment_ribbon/static/src/css/ribbon.css

```diff
@@ -16,7 +17,7 @@
     z-index: 1000;
     position: fixed;
     box-shadow: 0 0 3px rgba(0,0,0,.3);
-    background: rgba(255, 0, 0, .6);;
+    background: rgba(255,0,0,.6);
```

Comment:
@lasley, this change was made by 8.0 contributors, but it appears here because we have brought 8.0 commits that are not in 9.0 yet.